pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1861,6 +1876,22 @@ let
         };
       };
 
+      randomfractalsinc.vscode-data-preview = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "Data Preview";
+          publisher = "Random Fractal Inc";
+          version = "2.3.0";
+          shar256 = "DaI1/6IMQ44HrBM9tc04DGZmdlLdk1lVVMkMCa6cMHA=";
```

```suggestion
          shar256 = "DaI1/6IMQ44HrBM9tc04DGZmdlLdk1lVVMkMCa6cMHA=";
```
This is not right. Should have been `sha256` and please include the full SRI. Or use hash.

---

fixed typo, shar to sha